image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -247,10 +261,54 @@
   {{ $_ := set $._rox.scanner "slimDBImage" ._rox.image.scannerDb }}
   {{ include "srox.scannerInit" (list $ $._rox.scanner) }}
 
+  {{ $imagePullSecretNames = append $imagePullSecretNames "stackrox-scanner" }}
```

Comment:
was this not needed before?

Reply:
It was. Due to the second scanner being there I decided to restructure the code. See https://github.com/stackrox/stackrox/pull/9128/files/3ba129ae4fc4a6db84adb7f11e16df87212ca9bf#diff-f2abac3c707241f570e85a8e33cbff045efd515ec0eeaae4ce83e7e994125cc0L252.